examples/jupyter/pangeo_forge.ipynb

```diff
@@ -0,0 +1,73 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0844d55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pan3d import DatasetBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c66ff554",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_path = 'https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/noaa-coastwatch-geopolar-sst-feedstock/noaa-coastwatch-geopolar-sst.zarr'\n",
+    "state = {\n",
+    "    'active_array': 'analysed_sst',\n",
+    "    'x_array': 'lat',\n",
+    "    'x_scale': 2,\n",
+    "    'y_array': 'lon',\n",
+    "    't_array': 'time',\n",
+    "    't_index': 5,\n",
+    "    'resolution': 0.1,\n",
+    "    'main_drawer': False,\n",
+    "}\n",
+    "builder = DatasetBuilder(dataset_path=dataset_path, state=state)\n",
+    "\n",
+    "# Show viewer in cell output\n",
+    "await builder.layout.ready\n",
+    "builder.viewer"
+   ]
```

Comment:
This is weird

```python
await builder.layout.ready
builder.viewer
```

I would assume the viewer is the layout so I would assume

```python
await builder.viewer.ready
builder.viewer
```

Reply:
Do you mean that the await statement can be removed?

Reply:
I mean why do you have 2 things that are the same but with different name. It gets confusing. Stick with a single name.

Reply:
Oh I see now, they looked similar enough that I missed the diff

Reply:
Changed in [d65129e](https://github.com/Kitware/pan3d/pull/28/commits/d65129e719ede4e0114ca7b3be73100391ff2017)